contracts/PledgeAgent.sol

```diff
@@ -118,11 +121,11 @@ contract PledgeAgent is IPledgeAgent, System, IParamSubscriber {
   /// @param target Address of the target candidate
   error SameCandidate(address source, address target);
 
-  function init() external onlyNotInit {
+  constructor(Registry registry, uint256 powerBlockFactor_) System(registry) { //@openissue on mainnet: powerBlockFactor_ = 1e18
```

Comment:
@openissue indicates a location in the code that needs to be resolved.
Resolving may involve an investigation that determines that there's no real issue here, in which case @openissue may be replaced with a comment. It may also determine that a fix is needed, in which case the code should be changed.
Regardless - before deploying onto mainnet the @openissue count should be zero